packages/sources/wootrade/README.md

```diff
@@ -0,0 +1,55 @@
+# Chainlink External Adapter for Wootrade
+
+Adapter using the public Wootrade market API for both HTTP(s) and WS.
+
+### Environment Variables
+
+| Required? |  Name   |                                                        Description                                                         | Options | Defaults to |
+| :-------: | :-----: | :------------------------------------------------------------------------------------------------------------------------: | :-----: | :---------: |
+|     ✅      | API_KEY | An key to use the wootrade WS API |         |             |
```

Comment:
Can we remove that the API_KEY is required here, and rather have a WebSocket section below where we explain that it's required?